src/utils/resources/template/hooks/useVmTemplates.tsx

```diff
@@ -35,10 +39,16 @@ export const useVmTemplates = (namespace?: string): useVmTemplatesValues => {
 
   // Bug fix: TODO open BZ for SSP oc get projects don't show on projects when non-priv users
   const projectNames = projects?.map((proj) => proj?.metadata?.name);
-  if (projectNames && !projectNames.includes('openshift')) {
-    projectNames.push('openshift');
+  if (projectNames && !projectNames.includes(OPENSHIFT_NS)) {
+    projectNames.push(OPENSHIFT_NS);
   }
 
+  // Bug fix: the project with the images can't access templates which cause
+  // the template list view to show error instead of existing templates
```

Comment:
the correct fix should be not to fail if can't access templates for this namespace,
with correct fix if user can list templates from this namespaces they will not see them

Reply:
once we try to fetch templates from namespace `openshift-virtualization-os-images`/`kubevirt-os-images` we would fail, that is the behavior from the backend as well but this is a template we can list when list for projects, that is the reason I removed that namespace (`openshift-virtualization-os-images`/`kubevirt-os-images`). 
do you have another suggestion how to go around it?